src/python/grpcio_tests/tests_aio/unit/_test_server.py

```diff
@@ -42,9 +41,14 @@
 
 class _TestServiceServicer(test_pb2_grpc.TestServiceServicer):
 
-    async def UnaryCall(self, unused_request, context):
+    async def UnaryCall(self, request, context):
```

Comment:
Correct me if I'm wrong, but this sever is being used in the interop tests and also in some of the unit tests. Would it be worth to - in the short term - start stoping use the interop test server for the unit test? Otherwise we had the risk to have friction between both environments.

IMO we must follow what you already did, having ad-hoc servers in the unit test - placed in the same file - and use this server exclusively for the interop test (which would mean that the file could be moved under the interop folder).

WDYT?

Reply:
The interop server has kind of strict spec that will fail the interop test if we don't follow. Also, the spec has been implemented in many more languages. Personally, I believe reuse a stable common test server won't hurt, if the test case doesn't require any additional modification for the test server. E.g. like the metadata test.

For tests more complex than that, I agree we should use ad-hoc servers.